src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/PhysicalMarker/MarkerDetectorCoordinateService.cs

```diff
@@ -183,20 +183,30 @@ private async Task OnDiscoverCoordinatesImplAsync(CancellationToken cancellation
             }
         }
 
+#if UNITY_EDITOR
         private async Task OnDiscoverCoordinatesEditorAsync(CancellationToken cancellationToken, int[] idsToLocate = null)
         {
-            if (idsToLocate == null ||
-                idsToLocate.Length != 1)
+            if (UnityCompositorInterface.IsArUcoMarkerDetectorSupported())
             {
```

Comment:
do we need a try catch around UnityCompositorInterface calls? What happens for mobile simulation that executes in this code path if hte compositor dll doesn't exist locally?